content/en/apps/reference/app-settings/transitions.md

```diff
@@ -352,24 +354,47 @@ No custom configuration for `generate_patient_id_on_people`.
 
-Unmuting action:
+Server-side:
 
-- updates target contact's topmost muted ancestor<sup>[1][5]</sup> and all its descendants, removing the `muted` property
+- updates the target contact's topmost muted ancestor<sup>[1][5]</sup> and all its descendants, removing the `muted` property
 - adds a `muting_history` entry to Sentinel `info` docs for every updated contact<sup>[7]</sup>
 - updates all connected registrations<sup>[3]</sup>, changing the state of all present/future<sup>[6]</sup> `muted` `scheduled_tasks` to `scheduled`
-
-[1] Contacts that are already in the correct state are skipped. This applies to updates to the contact itself, updates to the Sentinel `muting_history` and to the connected registrations (registrations of a contact that is already in the correct state will not be updated).
-[2] The date represents the moment Sentinel has processed the muting action
-[3] target contact and descendants' registrations
-[4] `scheduled_tasks` being in either `scheduled` or `pending` state
-[5] Because the muted state is inherited, unmuting cascades upwards to the highest level muted ancestor. If none of the ancestors is muted, unmuting cascades downwards only.
-[6] `scheduled_tasks` which are due today or in the future. All `scheduled_tasks` with a due date in the past will remain unchanged.
+- as of 3.12.0, updates the contact's client-side `muting_history` to set the `last_update` property to `server` and update the `server` section with the current date and muted state.
+- as of 3.12.0, if the report was processed on the client, all "following" muting/unmuting events that have affected the same contacts will be replayed. This means the transition _could_ end up running multiple times over the same report<sup>[10]</sup>.
+
+[1] Contacts that are already in the correct state are skipped. This applies to updates to the contact itself, updates to the Sentinel `muting_history` and to the connected registrations (registrations of a contact that is already in the correct state will not be updated).  
+[2] The date represents the moment Sentinel has processed the muting action  
+[3] target contact and descendants' registrations  
+[4] `scheduled_tasks` being in either `scheduled` or `pending` state  
+[5] Because the muted state is inherited, unmuting cascades upwards to the highest level muted ancestor. If none of the ancestors is muted, unmuting cascades downwards only.  
+[6] `scheduled_tasks` which are due today or in the future. All `scheduled_tasks` with a due date in the past will remain unchanged.  
+[8] The date represents the device's date when the report is processed.  
+[9] Replaying is required due to how PouchDB <-> CouchDB synchronization does not respect the order in which the documents have been created, to ensure that contacts end up in the correct muted state.    
+[10] The updated contacts are limited to the contacts available on the device.  
```

Comment:
I hadn't noticed our use of footnotes before, and find it quite hard to find/use given that they aren't cross linked (eg it isn't clear where to go to see the footnotes, and 7 is separate). It may be outside of the scope of this PR to fix, but just highlighting as we are adding to them here.

Reply:
Yea, I think with plain markdown files it wasn't possible to superscript and also link, maybe we have some options now.

Reply:
I'll create a separate issue for the footnotes.

Reply:
Closing the loop: https://github.com/medic/cht-docs/issues/509